Loadify/View/Components/CustomTextField.swift

```diff
@@ -36,8 +38,14 @@ struct CustomTextField: View {
                         
                     }.padding(.leading, 16)
                 }
-                
-                if !text.isEmpty {
+                if text.isEmpty {
+                    Button(action: onPaste, label: { // Use the onPaste closure here
```

Comment:
You are calling the `onPaste` Closure. Instead create a private function named `didTapOnPasteIcon`. Assign the `UIPasteboard.general.string` to the `@Binding` directly. Since it's a property wrapper you don't need mutating function. 

onTap of the button call the newly created `didTapPasteIcon` function. This will make our code much cleaner.